Tools/TestWebKitAPI/Tests/WebKitCocoa/WKAttachmentTests.mm

```diff
@@ -2590,6 +2602,22 @@ static void _generateBestRepresentationForRequest(id, SEL)
     EXPECT_WK_STREQ("text/vcard", info.contentType);
 }
 
+TEST(WKAttachmentTestsIOS, InsertDroppedCalendarInviteAsAttachment)
+{
+    auto itemProvider = calendarInviteForTesting();
+    auto webView = webViewForTestingAttachments();
+    auto simulator = adoptNS([[DragAndDropSimulator alloc] initWithWebView:webView.get()]);
+    [simulator setExternalItemProviders:@[ itemProvider.get() ]];
+    [simulator runFrom:CGPointMake(25, 25) to:CGPointMake(100, 100)];
+
+    [webView expectElementCount:1 querySelector:@"ATTACHMENT"];
+    EXPECT_WK_STREQ("text/calendar", [webView valueOfAttribute:@"type" forQuerySelector:@"attachment"]);
+    EXPECT_EQ(1U, [simulator insertedAttachments].count);
+    _WKAttachmentInfo *info = [simulator insertedAttachments].firstObject.info;
+    EXPECT_TRUE([info.name containsString:@".ics"]);
```

Comment:
Doesn't matter for the test, but I'm curious: What is the inserted file name?

Reply:
Currently, an (otherwise-unnamed) calendar event gets dropped as `"ICS File.ics"`.